Source/MonoGame.Extended/TextureAtlases/TextureAtlas.cs

```diff
@@ -119,5 +121,42 @@ public static TextureAtlas Create(Texture2D texture, int regionWidth, int region
 
             return textureAtlas;
         }
+
+        public static TextureAtlas Create(Texture2D texture, int framewidth, int frameheight, int rows, int cols, Vector2 startpoint = Vector2.Zero, int rowborder = 0, int colborder = 0, int frameCount = -1, bool lefttop = true)
```

Comment:
One odd issue with the default value for startpoint. It didn't compile in Visual Studio. Does it compile for you?